02_working_with_strings/translate.py

```diff
@@ -1,16 +1,17 @@
-# Copyright (c) 2015 Ami. All rights reserved.
+# Copyright (c) 2016 Ami. All rights reserved.
 
 def user_input():
     response = input("Enter some text: ")
-    translate(response.lower())
+    return response
 
 def translate(text):
     result = text.replace("a", "4").replace("b", "8").replace("e", "3").replace("i", "1").replace("o", "0").replace("s", "5").replace("t", "7")
```

Comment:
This line is too long, break it to several lines.
On second thought, why don't you create a map object in the form of:
`dictionary = [{"term": "a", "translation": "4"}, {"value": ...}]`
and then traverse it using a loop to replace all terms in the dictionary more elegantly?